geode-apis-compatible-with-redis/src/distributedTest/java/org/apache/geode/redis/SSLDUnitTest.java

```diff
@@ -87,8 +87,9 @@ public void givenMutualAuthentication_clientErrorsWithoutKeystore() throws Excep
 
     // Create the client without a keystore
     try (Jedis jedis = createClient(false, false)) {
-      assertThatThrownBy(jedis::ping)
-          .hasMessageContaining("readHandshakeRecord");
+      assertThatThrownBy(jedis::ping).satisfiesAnyOf(
+          e -> assertThat(e.getMessage()).contains("SSLException"),
+          e -> assertThat(e.getMessage()).contains("SSLHandshakeException"));
```

Comment:
These assertions could be combined to a single one:
```
assertThatThrownBy(jedis::ping).isInstanceOf(SSLException.class);
```
Since `SSLHandshakeException` extends `SSLException`. Alternately, you could use `satisfiesAnyOf()` with assertions on the message body rather than exception name, but this might be trickier if you don't know all the different messages that might be expected in exceptions here.

Reply:
I've removed this block since exceptions are now just coming from the `createClient` call.

Reply:
And... now I've added it back since it seems that the `createClient` can also sometime fail (I have a feeling it's platform or JDK specific). But if the client is created, then this ping will definitely fail.

Reply:
The top level exception is always `JedisConnectionException` and the ones here are somewhere in the stack but not even the root cause. That's why I'm just looking at the message since that always contains one of these. I also have a strong suspicion that different plaforms and/or JDK produce slightly different messages or exception stacks. This seems to be one way to cover all bases.